packages/kuma-gui/src/app/meshes/views/MeshListView.vue

```diff
@@ -27,6 +27,7 @@
         </template>
 
         <XI18n
+          v-if="can('view growth-new-empty-states') && data?.items.length === 0 ? false : true"
```

Comment:
Since `data?.items.length === 0` already returns a `boolean` I'd change it slightly to

```suggestion
          v-if="can('view growth-new-empty-states') && data?.items.length > 0"
```

or even shorter with `undefined` or `0` being `falsy`

```suggestion
          v-if="can('view growth-new-empty-states') && data?.items.length"
```